docs/misc/changelog.rst

```diff
@@ -20,6 +20,8 @@ Pre-Release 2.5.1a0 (WIP)
   ``set_attr`` now returns ``None`` rather than a list of ``None``. (@kantneel)
 - ``GAIL``: ``gail.dataset.ExpertDataset` supports loading from memory rather than file, and
   ``gail.dataset.record_expert`` supports returning in-memory rather than saving to file.
+- fixed bug where result plotter would crash on very short runs
```

Comment:
Please add your username at the end of the entry (see above), and add it at the bottom to the list of contributors ;)